src/main/java/com/github/dockerjava/core/util/CertificateUtils.java

```diff
@@ -32,12 +32,13 @@ private CertificateUtils() {
 
     public static boolean verifyCertificatesExist(String dockerCertPath) {
```

Comment:
Method is static so you can place in `src/test/resource/$package/$class/method` 3 files and call this method with test path, should be pretty simple.